docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -1,432 +1,391 @@
 
-InnoDB supports [data encryption for InnoDB tables] stored in file-per-table tablespaces. This feature provides an at-rest encryption for physical tablespace data files.
+Percona XtraBackup supports two distinct types of encryption:
 
-For an authenticated user or an application to access an encrypted F tablespace, InnoDB uses the master encryption key to decrypt the tablespace key. The master encryption key is stored in a keyring. 
+* [InnoDB tablespace encryption](#innodb-tablespace-encryption): The MySQL server handles encryption using keyring components. InnoDB tablespace encryption provides at-rest encryption for physical tablespace data files at the database level.
 
-Percona XtraBackup supports the following keyring components which are stored in the `plugin` directory: 
+* [Backup-level encryption](#backup-level-encryption): XtraBackup applies an additional encryption layer during the backup process, independent of InnoDB encryption. Backup-level encryption protects backup files themselves.
 
-* [keyring_vault](#use-the-keyring-vault-component) component
+These encryption types are independent. You can use them separately or together for enhanced security.
 
-* [keyring_file](#use-the-keyring-file-component) component
+## InnoDB tablespace encryption
 
-* [Key Management Interoperability Protocol (KMIP)]
+InnoDB supports [data encryption for InnoDB tables] in file-per-table tablespaces. When accessing an encrypted tablespace, InnoDB uses the master encryption key to decrypt the tablespace key. InnoDB stores the master encryption key in a keyring.
+
+Percona XtraBackup supports the following keyring components:
 
+* [keyring_vault](#use-the-keyring-vault-component)
+* [keyring_file](#use-the-keyring-file-component)
+* [Key Management Interoperability Protocol (KMIP)]
 * [Amazon Key Management Service (AWS KMS)]
 
-Percona XtraBackup {{vers}} and later versions do not support any plugin versions of the security features.
+Percona XtraBackup {{vers}} and later versions support only component versions of the security features.
+
+### Configuration file name requirements
+
+When you use the [`--component-keyring-config`](xtrabackup-option-reference.md#component-keyring-config) option, the configuration file name must match exactly the expected name for each keyring component. If the file name does not match exactly, XtraBackup ignores the option.
+
+| Component | Expected config-file name |
+|-----------|---------------------------|
+| Vault | `component_keyring_vault.cnf` |
+| KMS | `component_keyring_kms.cnf` |
+| KMIP | `component_keyring_kmip.cnf` |
+| File | `component_keyring_file.cnf` |
+
+!!! warning "File-name restriction"
+
+    The configuration file name must match exactly the expected name for your keyring component. See [Configuration file name requirements](#configuration-file-name-requirements) for the exact file names required. This requirement applies to all prepare and restore operations.
```

Comment:
We dont need them for taking backups. Only for prepare phse. So lets move this to under prepare section.